redis/src/types.rs

```diff
@@ -121,9 +120,23 @@ pub enum Value {
     /// to express nested structures.
     Bulk(Vec<Value>),
     /// A status response.
-    Status(String),
+    Status(String), // maybe change to SimpleString ?
     /// A status response which represents the string "OK".
     Okay,
+    /// Ordered map value from the server. use `as_map_iter` function.
+    Map(Vec<Value>),
+    /// Unordered set value from the server.
+    Set(Vec<Value>),
+    /// A floating number response from the server.
+    Double(f64),
+    /// A boolean response from the server.
+    Boolean(bool),
+    /// First String is format and other is the string
+    VerbatimString(String, String),
+    /// Very large number that out of the range of the signed 64 bit numbers
+    BigNumber(String),
```

Comment:
IMO [BigInt](https://crates.io/crates/num-bigint) is more fitting here.

Reply:
I think using resp3 names is better

Reply:
Sorry, I wasn't clear - I meant that instead of returning a string, you should return a BigInt type, from the BigInt crate.